spec/acceptance/realtime/channel_history_spec.rb

```diff
@@ -184,8 +184,11 @@ def ensure_message_history_direction_and_paging_is_correct(direction)
       end
 
       context 'when channel receives update event after an attachment' do
```

Comment:
What spec point is this test for? I don't know what "when channel recieves update event" means. The only spec point that describes how `attach_serial` should be updated is RTL15a, which just says that

>  attachSerial is unset when the channel is instantiated, and is updated with the channelSerial from each ATTACHED ProtocolMessage received from Ably with a matching channel attribute

So if this is a test for RTL15a, then I don't understand why the test does all this stuff like `rest_channel.publish event, message_before_attach`…

Reply:
Generally, `channelSerial` has a format `xxxx-xxx-0`, then last digit is incremented for subsequent serials. Currently, we are setting it to `1` from `0`.
Maybe, I will update the test so it's easy to understand 👍

Reply:
But the exact value of the `channelSerial` doesn't matter at all, does it?

Reply:
Okay, updated the code https://github.com/ably/ably-ruby/pull/421/commits/ba82881d667e842a3f10901504723806fdbeadc6